packages/dd-trace/src/noop/proxy.js

```diff
@@ -4,18 +4,21 @@ const NoopTracer = require('./tracer')
 const NoopAppsecSdk = require('../appsec/sdk/noop')
 const NoopDogStatsDClient = require('./dogstatsd')
 const NoopLLMObsSDK = require('../llmobs/noop')
+const NoopAIGuardSDK = require('../aiguard/noop')
 
 const noop = new NoopTracer()
 const noopAppsec = new NoopAppsecSdk()
 const noopDogStatsDClient = new NoopDogStatsDClient()
 const noopLLMObs = new NoopLLMObsSDK(noop)
+const noopAIGuard = new NoopAIGuardSDK(noop)
```

Comment:
as said in one of the comments above, i don't think passing the nooptracer is useful here
```suggestion
const noopAIGuard = new NoopAIGuardSDK()
```